README.md

```diff
@@ -19,17 +19,14 @@ it doesn't use the extension to check, the extension is merely used to specify t
 file type magic number to use.
 
 ```haskell
->>> magic <- decodedJSON
->>> scriptBool <- hasSignature (magic) "foo/myrichtext" "rtf"
->> isJust scriptBool && fromJust scriptBool
+>>> hasSignature "foo/myrichtext" "rtf"
```

Comment:
I switched the arguments, so this should be `>>> hasSignature "rtf" "foo/myrichtext"`

Reply:
Why not make it something that's always used infix so it always reads `"foo/myrichtext" `hasSignature` "rtf`"?

Reply:
It depends on what the goal is. Having the filepath as the second argument makes for nicer currying. Having it as the first argument makes it more readable when applying with infix. There's not really a "correct" way to do it, but I always appreciate it if functions curry logically.

Reply:
Ohhhhhh that does make a lot of sense. So you could make something like `hasRtfSignature`, right?

Reply:
Yes, for example. Or when filtering:
`filter (hasSignature "rtf") allFiles`